code/processes/wdb.q

```diff
@@ -516,7 +477,7 @@ startup:{[]
 	.lg.o[`init;"partition has been set to [savedir]/[", (string partitiontype),"]/[tablename]/", $[writedownmode~`partbyattr;"[parted column(s)]/";""]];
 	if[saveenabled;
 		//check if tickerplant is available and if not exit with error 
-		.servers.startupdepcycles[.wdb.tickerplanttypes;.wdb.tpconnsleepintv;.wdb.tpcheckcycles]; 
+		/-.servers.startupdepcycles[.wdb.tickerplanttypes;.wdb.tpconnsleepintv;.wdb.tpcheckcycles]; 
```

Comment:
remove if unneeded, but is it unneeded? possibly the condition for the `if` here needs to be changed/added to to prevent this running when we don't want it to, but I think in a typical non-finspace installation we probably still want this to run?